public/servicenow_for_mattermost_notifications_v2.2.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?><unload unload_date="2022-11-29 12:14:37">
+<?xml version="1.0" encoding="UTF-8"?><unload unload_date="2023-01-17 06:37:41">
```

Comment:
What changes have been done in this update set?

Reply:
this update set contains the `epic_phase_3` update set

Reply:
Why do we need to backport phase 3 update set to phase 2?

Reply:
this contains some extra scripted rest APIs for getting urgency and impact fields and table fields. but we have removed those fields from the modal. should we remove this change?